R/get_stats.R

```diff
@@ -201,7 +220,8 @@ getStatsIntervals = function(
   method = match.arg(method)
 
   statType = attr(posteriorStats, 'statType')
-  idCols = intersect(c('cond', 'feature', 'posterior_sample'),
+  conds = if(statType == 'rhy') 'cond' else c('cond_1', 'cond_2')
```

Comment:
This can't be right, and it's concerning that the tests passed.

Reply:
That's weird; I don't have that deletion on my local branch. Same with the `return(d)`.

Reply:
Nevermind, I misread the diff. The thing about `return(d)` in the next comment is still true, though.